app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -45,6 +47,24 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       bindToggleForSubsections();
       bindToggleOpenCloseAllButton();
 
+      // When navigating back in browser history to the accordion, the browser will try to be "clever" and return
```

Comment:
It would. I think ideally we'd restore all the expanded sections again (like we did when using session storage). The problem with session storage was that if a user followed a circular journey that indirectly brought them back to the accordion, we found that users in testing were very confused as to why some sections were already open.

The problem of detecting if a user has just hit "Back" in their browser vs. (say) loading a bookmark is quite hard, but I'm open to suggestions.